cyral/resource_cyral_repository.go

```diff
@@ -89,18 +100,48 @@ func resourceRepository() *schema.Resource {
 				Required:    true,
 			},
 			"name": {
-				Description: "Repository name that will be used internally in the control plane (ex: `your_repo_name`)",
+				Description: "Repository name that will be used internally in the control plane (ex: `your_repo_name`).",
 				Type:        schema.TypeString,
 				Required:    true,
 			},
 			"labels": {
-				Description: "labels enable you to categorize your repository",
+				Description: "Labels enable you to categorize your repository.",
 				Type:        schema.TypeList,
 				Optional:    true,
 				Elem: &schema.Schema{
 					Type: schema.TypeString,
 				},
 			},
+			// 'advanced' is equivalent to 'properties' in the Cyral
+			// v1/repos API, but with a user-friendly name.
+			"advanced": {
+				Description: "Contains advanced repository configuration.",
+				Type:        schema.TypeSet,
+				Optional:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"replica_set": {
```

Comment:
```suggestion
						"mongodb_replica_set": {
```

Reply:
I see. Thanks! I got a bit confused as to the level of generality we were aiming with the 'replica set' concept. This suggestion makes it easier for the MongoDB user to find the parameters he needs.